packages/grafana-data/src/events/common.ts

```diff
@@ -0,0 +1,45 @@
+import { DataFrame } from '../types';
+import { eventFactory } from './eventFactory';
+import { BusEventWithPayload } from './types';
+
+/** @alpha */
+export interface DataHoverPayload {
```

Comment:
@torkelo I think this is close to a reasonable payload for the hover event.  It would allow multi-axis hover and a consistent way to to hover a series row/column.